pkg/cmd/util.go

```diff
@@ -56,3 +58,40 @@ func getApplicationsFromFiles(files []string) ([]inputData, error) {
 	}
 	return appData, nil
 }
+
+// GetAllYMLFiles if path in argument is directory get all *.yml and *.yaml files
+// in that directory. If path is file just add it to output list as it is.
+func GetAllYMLFiles(paths []string) ([]string, error) {
+	var files []string
+
+	for _, path := range paths {
+		fileInfo, err := os.Stat(path)
+		if err != nil {
+			return nil, errors.Wrapf(err, "can't get file info about %s", path)
+		}
+		if fileInfo.IsDir() {
+			ymlFiles, err := filepath.Glob(filepath.Join(path, "*.yml"))
+			if err != nil {
+				return nil, errors.Wrapf(err, "can't list *.yml files in %s", path)
+			}
+			files = append(files, ymlFiles...)
+
+			yamlFiles, err := filepath.Glob(filepath.Join(path, "*.yaml"))
```

Comment:
We are having to run `filepath.Glob` twice, once for yaml and one for yml. Can we use some pattern matching to make it one?
I tried `ls *.y{a,}ml` for the ls command, and it worked there, can we have something similar here?